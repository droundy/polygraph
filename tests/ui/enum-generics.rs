polygraph::schema!{
    type Database;
    pub enum MyEnum<T> {
        Hello(T),
        Goodbye,
    }
}

fn main() {
}
