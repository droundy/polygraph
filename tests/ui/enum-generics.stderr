error: schema! does not support generic types.
 --> $DIR/enum-generics.rs:3:20
  |
3 |     pub enum MyEnum<T> {
  |                    ^^^
